airbyte-integrations/connectors/source-asana/source_asana/manifest.yaml

```diff
@@ -0,0 +1,2622 @@
+              partition_field: "users_gid"
+
+  portfolios_compact_partition_router:
+    type: SubstreamPartitionRouter
+    parent_stream_configs:
+      - stream: "#/definitions/portfolios_compact_stream"
+        parent_key: "gid"
+        partition_field: "portfolios_gid"
+
+  portfolios_stream:
+    $ref: "#/definitions/base_stream"
+    name: "portfolios"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/portfolios_schema"
+    $parameters:
+      path: "/portfolios/{{ stream_slice['portfolios_gid'] }}"
+    retriever:
+      $ref: "#/definitions/retriever"
+      paginator:
+        $ref: "#/definitions/paginator_base"
+      partition_router:
+        $ref: "#/definitions/portfolios_compact_partition_router"
+
+  portfolios_memberships_stream:
+    $ref: "#/definitions/base_stream"
+    name: "portfolios_memberships"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/portfolios_memberships_schema"
+    $parameters:
+      path: "/portfolio_memberships"
+    retriever:
+      $ref: "#/definitions/retriever"
+      requester:
+        $ref: "#/definitions/requester"
+        request_parameters:
+          portfolio: "{{ stream_slice['portfolios_gid'] }}"
+      paginator:
+        $ref: "#/definitions/paginator_base"
+      partition_router:
+        $ref: "#/definitions/portfolios_compact_partition_router"
+
+  custom_fields_stream:
+    $ref: "#/definitions/base_stream"
+    name: "custom_fields"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/custom_fields_schema"
+    $parameters:
+      path: "/workspaces/{{ stream_slice['workspace_gid'] }}/custom_fields"
+    retriever:
+      $ref: "#/definitions/retriever"
+      paginator:
+        $ref: "#/definitions/paginator_base"
+      partition_router:
+        $ref: "#/definitions/workspaces_partition_router"
+
+  organization_exports_stream:
+    $ref: "#/definitions/base_stream"
+    name: "organization_exports"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/organization_exports_schema"
+    $parameters:
+      path: "/organization_exports/{{ stream_slice.gid }}"
+    retriever:
+      $ref: "#/definitions/retriever"
+      paginator:
+        $ref: "#/definitions/paginator_base"
+      partition_router:
+        - type: ListPartitionRouter
+          values: "{{ config['organization_export_ids' ] }}"
+          cursor_field: gid
+
+  events_stream:
+    $ref: "#/definitions/base_stream"
+    name: "events"
+    primary_key: "created_at"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/events_schema"
+    $parameters:
+      path: "/events"
+    retriever:
+      $ref: "#/definitions/retriever"
+      paginator:
+        $ref: "#/definitions/token_paginator_base"
+      requester:
+        $ref: "#/definitions/requester"
+        request_parameters:
```

Comment:
Missing request parameter: `opt_fields`. Please add it according to the original implementation. Also, please pay attention how the HTTP code 412 should be handled [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-asana/source_asana/streams.py#L218)